.github/workflows/cd.yml

```
@@ -2,7 +2,7 @@ name: CD
 
 on:
   push:
-    branches: [develop]
+    branches: [ develop ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)